src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2758,7 +2758,6 @@ namespace winrt::TerminalApp::implementation
     // - args: the arguments specifying how to set the display status to ShowWindow for our window handle
     winrt::fire_and_forget TerminalPage::_ShowWindowChangedHandler(const IInspectable /*sender*/, const Microsoft::Terminal::Control::ShowWindowArgs args)
     {
-        co_await resume_foreground(Dispatcher());
```

Comment:
With the `if` check for `_showHideWindowThrottler` removed, we can safely drop this `resume_foreground`. It's also a nice safety fix, because it doesn't hold onto `this` strongly.